src/raftstore/store/fsm/apply.rs

```diff
@@ -1942,6 +1949,24 @@ impl ApplyDelegate {
         ))
     }
 
+    fn exec_mvcc_gc(
```

Comment:
I don't think it's a good idea to leak txn concepts to raft layer. How about inspecting the deleted keys and sending `MvccGcTask` in `raftstore::coprocessor`?

Reply:
I think it's not raft layer, it's just a proposal that needs to do some special things for it. We also have `ComputeHash` like this.